.evergreen/config.yml

```diff
@@ -605,8 +621,11 @@ functions:
   run-valid-ocsp-server:
     - command: shell.exec
       params:
+        shell: "bash"
         script: |
           cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+          . ./activate-ocspvenv.sh
+
           ${PYTHON3_BINARY} -m venv ./venv
           ./venv/${VENV_BIN_DIR|bin}/pip3 install -r mock-ocsp-responder-requirements.txt
```

Comment:
```suggestion
```

`activate-ocspvenv.sh` creates the virtual environment and installs dependencies.